GTBitsOfGood/the-ray
# Added mobile view for stats page and touched up desktop view

## WheelRight Stats Mobile View

Issue Number(s): #96 

### What does this PR change and why?

Adds mobile view for the wheelright statistics slides so the site doesn't look bad on phones.

Before Desktop View:
<img width="1680" alt="Screen Shot 2022-09-11 at 8 32 37 PM" src="https://user-images.githubusercontent.com/14205166/189556095-174f3f02-5a18-4fdc-baa6-ae193e134f8b.png">

After Desktop View:
<img width="1680" alt="Screen Shot 2022-09-11 at 8 34 13 PM" src="https://user-images.githubusercontent.com/14205166/189556160-6ec13c47-1b5c-45d6-b45c-fff370731578.png">

Before Mobile View:
<img width="407" alt="Screen Shot 2022-09-11 at 8 33 19 PM" src="https://user-images.githubusercontent.com/14205166/189556125-7cba07a2-257d-4657-bf56-f3130dea8e8f.png">

After Mobile View:
<img width="406" alt="Screen Shot 2022-09-11 at 8 33 46 PM" src="https://user-images.githubusercontent.com/14205166/189556149-bb6e4b27-1c35-4653-9f02-ea87d9c45c7e.png">

### Checklist


### Critical Changes

- Adds screensize checking to Stats.css

### Related PRs

- #111 (This PR Replaces PR 111)

### How to Test

1. Run the site with `npm start`
2. Select WheelRight
3. Scroll to Statistics Pages
4. Use developer tools Responsive mode to change the screen size and observe the application
